site/_data/deprecationCalendar.js

```diff
@@ -0,0 +1,73 @@
+const path = require('path');
```

Comment:
Move into `site/_data/deprecation-calendar/index.js` - not too sure if that works, but it would be neat 😄

Reply:
Uhm - tried this and technically works, the issue is that then the data gets formatted in a slightly different way. Instead of having just {myData} I would get something like { articles: {articlesData}, index: {myData}.

Which happy to go that way, I suppose it makes it cleaner on file system, but a tiny bit less clean when the data is needed.

@matthiasrohmer